static/src/javascripts/projects/commercial/modules/carrot-slot.js

```diff
@@ -3,6 +3,7 @@ import fastdom from 'lib/fastdom-promise';
 import { addSlot } from 'commercial/modules/dfp/add-slot';
```

Comment:
@regiskuckaertz I've rewritten it to use the spacefinder engine/flow. Anything I've missed or glossed over here?